scala/README.md

```diff
@@ -444,7 +501,7 @@ It is definitely recommended to read the full Twitter's "[Effective Scala]" guid
     def findFemales: Seq[People]
     ```
 
-0. Do not define abstract `val`s in traits and abstract classes. Abstract `val`s are a source of headaches and unexpected behavior in Scala:
+0. Do not define abstract `val`s in traits or classes. Abstract `val`s are a source of headaches and unexpected behavior in Scala:
```

Comment:
"or" is a good change, but I'd keep "abstract" here. You shouldn't define an abstract val in a class... because you can't :)